listenbrainz/webserver/static/js/src/BrainzPlayer.tsx

```diff
@@ -273,11 +273,13 @@ export default class BrainzPlayer extends React.Component<
     if (currentListenIndex === -1) {
       nextListenIndex = direction === "up" ? listens.length - 1 : 0;
     } else if (direction === "up") {
-      nextListenIndex =
-        invert === true ? currentListenIndex + 1 : currentListenIndex - 1 || 0;
+      nextListenIndex = invert
+        ? currentListenIndex + 1
+        : currentListenIndex - 1 || 0;
     } else if (direction === "down") {
-      nextListenIndex =
-        invert === true ? currentListenIndex - 1 || 0 : currentListenIndex + 1;
+      nextListenIndex = invert
+        ? currentListenIndex - 1 || 0
+        : currentListenIndex + 1;
```

Comment:
This is incorrect and will break playback. See #1759

Generally, this section of BP has been refactored in #1728 and #1759 and shouldn't be modified before we merge the changes to avoid conflicts.

Reply:
I see. Thanks! Fixed.